src/Cake.SqlServer/SqlServerAliases.cs

```diff
@@ -162,19 +163,15 @@ public static void DropAndCreateDatabase(this ICakeContext context, String conne
         /// Execute any SQL command.
         /// </summary>
         /// <param name="context">The Cake context.</param>
-        /// <param name="connectionString">Connection string where to connect to. Connection script must specify what database to connect to</param>
+        /// <param name="connectionString">The connection string. You may want to specify Initial Catalog. If there are changing parameters, <see cref="SqlConnectionStringBuilder"/> is recommended to escape input.</param>
         /// <param name="sqlCommands">SQL to be executed</param>
         /// <example>
         /// <code>
         ///     #addin "nuget:?package=Cake.SqlServer"
         /// 
-        ///     Task("Database-Operations")
-        ///          .Does(() =>
-        ///          {
-        ///             var connectionString = @"Server=(LocalDb)\v12.0";
-        ///             var sqlCommand = "Create table [CakeTest].dbo.[CakeTestTable] (id int null)";
-        ///             ExecuteSqlCommand(connectionString, sqlCommand);
-        ///         });
+        ///     var connectionString = @"Data Source=(LocalDb)\v12.0;Initial Catalog=MyDatabase";

```

Comment:
Please wrap code sample in `Task("ReCreate-Database").Does(() => {/*sample*/})` - for consistency